pkg/apis/pipeline/v1/task_validation_test.go

```diff
@@ -686,27 +686,6 @@ func TestTaskSpecValidateError(t *testing.T) {
 			Message: fmt.Sprintf("The value type specified for these keys %v is invalid", []string{"key1"}),
 			Paths:   []string{"params.task.properties"},
 		},
-	}, {
-		name: "keys defined in properties are missed in default",
```

Comment:
let's keep this test case around but just have it result in a successful validation.

Reply:
Sounds great! Done. Thanks @lbernick

Reply:
I think you might not have pushed these changes?

Reply:
It's under the valid test cases called `valid params type explicit` and the param name is `myobjWithDefaultPartialKeys`.